backend/setup.go

```diff
@@ -88,16 +90,37 @@ func setupProfiler(pluginID string) {
 	}
 }
 
+func getTracerCustomAttributes(pluginID string) []attribute.KeyValue {
+	var customAttributes []attribute.KeyValue
+	// Add plugin id and version to custom attributes
+	// Try to get plugin version from build info
+	// If not available, fallback to environment variable
+	var pluginVersion string
+	buildInfo, err := build.GetBuildInfo()
+	if err != nil {
+		Logger.Debug("Failed to get build info", "error", err)
+	} else {
+		pluginVersion = buildInfo.Version
+	}
+	if pluginVersion == "" {
+		if pv, ok := os.LookupEnv(PluginVersionEnv); ok {
+			pluginVersion = pv
+		}
+	}
+	if pluginVersion != "" {
+		customAttributes = append([]attribute.KeyValue{semconv.ServiceVersionKey.String(pluginVersion)}, customAttributes...)
+	}
+	customAttributes = append([]attribute.KeyValue{semconv.ServiceNameKey.String(pluginID)}, customAttributes...)
```

Comment:
Nit. Would the following simplify defaulting plugin version to empty if not set?

```suggestion
	customAttributes = []attribute.KeyValue{
		semconv.ServiceVersionKey.String(pluginVersion),
		semconv.ServiceNameKey.String(pluginID),
	)
```

Reply:
Yes, with your suggested change it will set the service key to an empty string:

![image](https://github.com/grafana/grafana-plugin-sdk-go/assets/16373015/8d6b2a43-faaa-4815-acfd-fa34a19fe2f2)

I can go ahead and do this change if it's okay to have an empty string rather than no `service.version` attribute at all. WDYT?

Reply:
I have applied the suggestion :smiley: